src/test/java/com/aws/greengrass/telemetry/TelemetryAgentTest.java

```diff
@@ -129,9 +129,10 @@ void cleanUp() throws IOException, InterruptedException {
         TelemetryConfig.getInstance().closeContext();
         telemetryAgent.shutdown();
         ses.shutdownNow();
+        executorService.shutdownNow();
         context.close();
-        ses.awaitTermination(2, TimeUnit.SECONDS);
-        TestFeatureParameters.internalDisableTestingFeatureParameters();
```

Comment:
keep this so that even if a test fails we don't have the test feature thing messing up future tests.